galactic/distribution.yaml

```diff
@@ -3812,6 +3812,23 @@ repositories:
       url: https://github.com/KumarRobotics/ublox.git
       version: foxy-devel
     status: maintained
+  ublox_dgnss:
+    release:
+      packages:
+      - ublox_dgnss
+      - ublox_dgnss_node
+      - ublox_ubx_interfaces
+      - ublox_ubx_msgs
+      tags:
+        release: release/galactic/{package}/{version}
+      url: https://github.com/aussierobots/ublox_dgnss-release.git
+      version: 0.1.0-1
+    source:
+      type: git
+      url: https://github.com/aussierobots/ublox_dgnss.git
+      version: master
+    status: developed
+    status_description: functional against Ublox ZED-F9P
```

Comment:
Also, leave out the `status_description` for now.  This is typically only used when we deprecate packages.